tools/wpt/run.py

```diff
@@ -539,6 +539,10 @@ def install(self, venv, channel=None):
         raise NotImplementedError
 
     def setup_kwargs(self, kwargs):
+        if kwargs["flatpak"]:
```

Comment:
I don't love putting these things on the class itself. I'd rather have a `browser.set_options(**kwargs)` method that would be called here and pick the browser-specific settings out of the kwargs dict.